R/dependencies.R

```diff
@@ -0,0 +1,410 @@
+##' @title Get the dependencies for a given report from the database
+##'
+##' @param name the name of the report
+##' @param id the id of the report, if omitted, use the id of the latest report
+##' @param upstream A boolean indicating if we want to move up or down the tree
+##' @param con A connection to a database
+get_dependencies_db <- function(name, id, upstream, con, list_all = FALSE) {
```

Comment:
All exported orderly functions have `orderly_` as their name, and in general I do not like the use of `get` here. How about simply `orderly_dependencies` (or perhaps `orderly_report_dependencies`)

Reply:
oh, I see this is not actually an exported function - in which case I think these roxygen blocks all require `@noRd` or similar to prevent creation of `man` files